config/jetpack-cloud-horizon.json

```diff
@@ -60,6 +60,7 @@
 		"backup": true,
 		"jetpack-cloud": true,
 		"jetpack-cloud-agency-dashboard": true,
+		"jetpack-cloud-plugin-management": false,
```

Comment:
Let's make this one true since I believe this is the env behind calipso.live URLs.